README.md

```diff
@@ -93,8 +88,9 @@ supported build environments that are most likely to work.
 Rust currently needs about 1.5 GiB of RAM to build without swapping; if it hits
 swap, it will take a very long time to build.
 
-There is a lot more documentation in the [wiki].
+There is more advice about [hacking on Rust][hack] on the [wiki].
 
+[hack]: https://github.com/rust-lang/rust/wiki/Note-getting-started-developing-Rust
```

Comment:
we want to kill the wiki, so this isn't good. maybe the CONTRIBUTING.md link is the righ tcall here?